src/rewriter/README.md

```diff
@@ -94,17 +112,26 @@ processing are controlled by `rewriter.py` and `parser.py`.
 The symbols usable in `(<id> <expr>+)` can be found in `node.py`. The sorts
 available can be found in `mkrewrites.py`.
 
-
 ## Changes from Previous Works
 
 In comparison to
 [fmcad22](https://ieeexplore.ieee.org/abstract/document/10026573), we have
-removed the `let` and `cond` expressions in favour of more atomic rewrite rules.
-`let` is replaced by `def` which allow symbols to be shared across the
-condition, source, and target terms.
-
-We also deleted the `const` modifier since the evaluation of constant
-expressions should be handled elsewhere.
+removed the `let` expression.  `let` is replaced by `def` which allow symbols to
+be shared across the condition, source, and target terms.
+
+We also deleted the `const` modifier the rationale is that there are 3 reasons to use `:const`:
```

Comment:
"Compared to previous work on this topic..."

Maybe start with something like this since const is not part of the FMCAD paper.

Reply:
I also think the sentence is a bit confusing, let me try to rephrase.

Reply:
done

Reply:
It is already better the way you rephrased it now! I think adding a comma after "modifier" and saying "the rationale behind this is" would have helped as well.